src/lib/common/statistics.h

```diff
@@ -315,9 +316,9 @@ extern int noOfSimulatedNotifications;
 
 /* ****************************************************************************
 *
-* timingStatistics - 
+* renderTimingStatistics -
 */
-extern std::string timingStatistics(std::string indent, Format format, std::string apiVersion);
+extern std::string renderTimingStatistics(void);
```

Comment:
Yeah, that's was I was afraid of ...
As I said, I believe this particular request should come out human-readable. That's why I implemented it that way ...